app/bulk_resource/views.py

```diff
@@ -25,4 +48,6 @@ def upload():
 @bulk_resource.route('/review')
 @login_required
 def review():
-    return render_template('bulk_resource/review.html')
+    csv_container = CsvContainer.query.first()
```

Comment:
Yes, definitely. For now, I added a `TODO`. Something is broken with `current_user` at the moment...